packages/react-sdk/src/context.tsx

```diff
@@ -35,11 +36,21 @@ export const ReactSdkContext = createContext<
     propsByInstanceIdStore: ReadableAtom<PropsByInstanceId>;
     assetsStore: ReadableAtom<Assets>;
     pagesStore: ReadableAtom<Pages>;
+    dataSourceValuesStore: ReadableAtom<Map<DataSource["id"], unknown>>;
+    setState: (
```

Comment:
should it be just setState? it may be better when more specific? e.g. setInstanceState?

Reply:
setDataSourceValue?

Reply:
yeah, that's good